src/libs/kata-types/src/annotations/cri_containerd.rs

```diff
@@ -11,3 +11,14 @@ pub const SANDBOX: &str = "sandbox";
 pub const CONTAINER: &str = "container";
 
 pub const SANDBOX_ID_LABEL_KEY: &str = "io.kubernetes.cri.sandbox-id";
+
+// Ref: https://pkg.go.dev/github.com/containerd/containerd/pkg/cri/annotations
```

Comment:
For reference, it's better to include the version in the URL: https://pkg.go.dev/github.com/containerd/containerd@v1.6.7/pkg/cri/annotations

Reply:
Fixed, nice catch.